interface/cmd/new_dependencies.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"net/url"
+
+	"github.com/msh5/boy/interface/dependency"
+)
+
+const (
+	GitHubHost = "github.com"
+	GistHost   = "gist.github.com"
```

Comment:
No need to expose these variables.

```suggestion
	gitHubHost = "github.com"
	gistHost   = "gist.github.com"
```